java/source/demo/pom.xml

```diff
@@ -32,14 +32,29 @@
         </dependency>
         <dependency>
             <groupId>org.apache.arrow</groupId>
-            <artifactId>arrow-memory-unsafe</artifactId>
+            <artifactId>arrow-memory-netty</artifactId>
             <version>${arrow.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.arrow</groupId>
             <artifactId>arrow-algorithm</artifactId>
             <version>${arrow.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.arrow</groupId>
+            <artifactId>flight-core</artifactId>
+            <version>6.0.0</version>
```

Comment:
6.0.0?

Reply:
We shouldn't mix Arrow versions.

Reply:
Ok lets install that manually for our github ci

Reply:
Gihub CI configured to install flight core 7.0.0 manually